pom.xml

```diff
@@ -57,8 +55,25 @@
         <version.arm-oraclelinux-wls-cluster-deletenode>1.0.1</version.arm-oraclelinux-wls-cluster-deletenode>
         <!--  versions  end -->
 
+        <git.tag>main</git.tag>
+        <git.repo>weblogic-azure</git.repo>
+        <git.repo.owner>oracle</git.repo.owner>
+        <artifactsLocationBase>https://raw.githubusercontent.com/${git.repo.owner}/${git.repo}/${git.tag}/weblogic-azure-vm</artifactsLocationBase>
+
+        <module.basedir>${project.basedir}</module.basedir>
+        <template.validation.tests.directory>${module.basedir}/../arm-ttk/arm-ttk</template.validation.tests.directory>
```

Comment:
I happen to know that this property lets us resolve the arm-ttk to where it is actually checked out locally. Because the workflows check out the arm-ttk at this location and they, I suggest this property get moved to the section where you define the other GitHub actions workflows.